package.json

```diff
@@ -18,5 +18,6 @@
         "sortablejs": "^1.15.3",
         "tailwindcss": "^3.4.14",
         "vite": "^5.4.10"
-    }
+    },
+    "packageManager": "pnpm@9.15.0+sha512.76e2379760a4328ec4415815bcd6628dee727af3779aaa4c914e3944156c4299921a89f976381ee107d41f12cfa4b66681ca9c718f0668fa0831ed4c6d8ba56c"
```

Comment:
Is this committed in error?

Reply:
Yes, this was committed by mistake. I have removed it now.